src/ShareRoutes.jsx

```diff
@@ -94,3 +73,95 @@ export default function ShareRoutes({installPrefix, appPrefix}) {
     </Routes>
   )
 }
+
+
+/**
+ * Forward page from /share to /share/v/p per spect at:
+ *   https://github.com/bldrs-ai/Share/wiki/URL-Structure
+ * @param {string} appPrefix The install prefix, e.g. /share.
+ * @return {Object}
+ */
+function Forward({appPrefix}) {
+  const location = useLocation()
+  const navigate = useNavigate()
+
+  useEffect(() => {
+    if (location.pathname == appPrefix) {
+      const dest = appPrefix + '/v/p'
+      debug().log('ShareRoutes#useEffect[location]: forwarding to: ', dest)
+      navigate(dest)
+    }
+  }, [location, appPrefix, navigate])
+
+  return <Outlet />
+}
+
+
+/**
+ * Check if input is a url
+ * @param {input} input
+ * @return {boolean} return true if url is found
+ */
+export function looksLikeLink(input) {
+  assertDefined(input)
+  return input.includes('/') || input.endsWith('.ifc')
```

Comment:
do you think we should look out for any other signs of URL?

Reply:
I thought through a bunch of cases and came up with this, along with test cases.